### PR TITLE
New rubocop2

### PR DESCRIPTION
**Pull request by jreidinger**
Adapt to new rubocop version

### CODE REVIEW

File: bin/yupdate [hunk 1]
```diff
@@ -224,7 +224,7 @@ module YUpdate
       system("mount", "--make-private", origdir)
 
       system("mount", "-t", "overlay", "overlay", "-o", "lowerdir=#{dir},"\
-        "upperdir=#{upperdir},workdir=#{workdir}", dir)
+                                                        "upperdir=#{upperdir},workdir=#{workdir}", dir)
```
Comment (dgdavid) [inline bin/yupdate:227]: Is this really needed? Looks weird.

File: src/lib/installation/widgets/online_repos.rb [hunk 2]
```diff
@@ -1,3 +1,22 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2021 SUSE LLC
```
Comment (dgdavid) [inline src/lib/installation/widgets/online_repos.rb:2]: NP: It's 2022 already :) You can even use [2017-2022]

**Review by dgdavid — APPROVED** [review]
LGTM

File: bin/yupdate (hunk 1 above)
Comment (jreidinger) [reply bin/yupdate:227]: well, it was caused by that string split in previous line. I think original code is strange. I will fix it

File: src/lib/installation/widgets/online_repos.rb (hunk 2 above)
Comment (jreidinger) [reply src/lib/installation/widgets/online_repos.rb:2]: we should use century there, it is more stable :)

File: src/lib/installation/clients/inst_prepareprogress.rb [hunk 3]
```diff
@@ -76,8 +77,8 @@ def main
           {
             "name"        => "images",
             "description" => _("Deploying Images..."),
-            "value"       => 300_000, # just make it longer than inst_finish, TODO: better value later
-            "units"       => :kb
+            "value"       => 400, # just make it longer than inst_finish
```
Comment (dgdavid) [inline src/lib/installation/clients/inst_prepareprogress.rb:80]: What a difference :)

**Review by dgdavid — APPROVED** [review]

**Pull request merged**